### DISCUSSION TITLE
Where can I find documentation for Voiceliner?

### DISCUSSION THREAD

**Discussion by caseymandell** [maxkrieger/voiceliner#76] [Q&A] [answered]
Where can I find documentation for Voiceliner?

**Comment by maxkrieger** [comment, marked as answer]
There is none sadly outside of the README - feel free to shoot questions here!

**Comment by caseymandell** [comment]
Dear Max,

Thanks for your response!  I’m really excited about using VoiceLiner to capture quick notes when I’m not near my computer.

Looking around at the beginning of my search for information about VoiceLiner I did come across a file about formatting checkboxes and some other functionalities.  That’s what I’m trying to find again.  Does the Readme file you’re referring to include that extra information?  I’ve looked through all the Readme files I can find.  I wonder if you could tell me the url of the Readme file you’re referring to?  Or even better yet paste it into the body of an email message?

Thanks again!
Sincerely,
Casey

From: Max Krieger ***@***.***>
Reply-To: maxkrieger/voiceliner ***@***.***>
Date: Monday, April 25, 2022 at 1:51 PM
To: maxkrieger/voiceliner ***@***.***>
Cc: Casey Mandell ***@***.***>, Author ***@***.***>
Subject: Re: [maxkrieger/voiceliner] Where can I find documentation for Voiceliner? (Discussion #76)


There is none sadly outside of the README - feel free to shoot questions here!

**Reply by maxkrieger** [reply]
Readme is here - https://github.com/maxkrieger/voiceliner#readme

I'm not exactly sure which file you're thinking of - there have been plenty of issue discussions on how checkboxes should be formatted on the issues tab though.
